test/output/3.sh.txt

```diff
@@ -0,0 +1,27 @@
+#!/bin/bash
+
+ff()
+{
+  while read FILE; do
+    echo "$FILE"
+
+  done <<END
+	$(list "$@")
```

Comment:
The TAB in here-document is untouched. I think that is the right thing.
And note that bashbeauty tries not to touch here-document otherwise it 
may change the input generated from that code.

Cheers,
Kyeong


On 18/06/16 00:01, Herman Bergwerf wrote:
> In test/output/3.sh.txt
> <https://github.com/hermanbergwerf/bashbeautify/pull/3#discussion_r67499576>:
>
>> @@ -0,0 +1,27 @@
>> +#!/bin/bash
>> +
>> +ff()
>> +{
>> +  while read FILE; do
>> +    echo "$FILE"
>> +
>> +  done <<END
>> +	$(list "$@")
>
> I think this formatting is actually not valid. There is still a tab
> character instead of spaces.
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub
> <https://github.com/hermanbergwerf/bashbeautify/pull/3/files/90a3c5228a0ad599630a1ac26fee823357316e85#r67499576>,
> or mute the thread
> <https://github.com/notifications/unsubscribe/ATBnfaGrJAqRTgQjZ9pR-fvQljnnWFykks5qMoy1gaJpZM4I2N9T>.
>